solr/test-framework/src/java/org/apache/solr/cloud/AbstractZkTestCase.java

```diff
@@ -65,7 +65,7 @@ public static void azt_beforeClass() throws Exception {
   }
 
   @AfterClass
-  public static void azt_afterClass() throws Exception {
```

Comment:
minor: probably has your attention because of the underscore but you could simply change to `aztAfterClass`.  The author likely wanted to disambiguate on a common base class -- reasonable.